### PR TITLE
uploading crop summary and stories with knobs

### PR DESCRIPTION
**Pull request by zenkaeser**
Make changes to the props and implement children for the content. Added withknobs for the status, stage name and other props for the summary info.
As you may see, I added everything in one place that is under Card and Crop Summary folders. 
Thanks.

### CODE REVIEW

File: src/client/components/StatusCard/status-card.component.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Card from '../card/card.component';
+import Card from '../Card/card.component';
```
Comment (zkwsk) [inline src/client/components/StatusCard/status-card.component.js]: Why are you changing this? The folder is lower case. Please make sure that files and paths are lower case throughout.

File: src/client/components/card/CropSummary/Summary.component.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (zenkaeser) [reply src/client/components/card/CropSummary/Summary.component.js]: Okay zaki, I will review them all and make a PR again

[the remark this answers is not in the document]
Comment (zenkaeser) [reply src/client/components/card/CropSummary/Summary.component.js]: can I delete this PR and make a new one instead?
Or I will just push another commit on top of this?Thanks

File: src/client/components/StatusCard/status-card.component.js (hunk 1 above)
Comment (zenkaeser) [reply src/client/components/StatusCard/status-card.component.js]: Sorry zaki, i got it confused with the Footer and StatusCard. I am now changing everything and conform with the configs set.

Comment (zenkaeser) [reply src/client/components/StatusCard/status-card.component.js]: They have different casing than the one on the card that is why I change it.

**Pull request closed without merging**